### PR TITLE
point to internal version of formatting guide

### PR DESCRIPTION
**Pull request by marcacohen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ using some simple formatting conventions. You can also author codelabs using mar
 This repo contains all the tools and documentation you’ll need for building and publishing
 your own codelabs.
 
-If you're interested in authoring codelabs, join [this group](https://groups.google.com/forum/#!forum/codelab-authors),
-which connects you with other authors and provides access to the [Codelab Formatting Guide](https://g.co/codelabs/guide).
+If you're interested in authoring codelabs, create a doc following the [Codelab Formatting Guide](FORMAT-GUIDE.md). 
+and see the [claat](claat) directory for a detailed description of the `claat` command line tool.
```
Comment (nicolasgarnier) [inline README.md]: We could point to the example doc that you have created here.

File: README.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ using some simple formatting conventions. You can also author codelabs using mar
...
 
-See the [claat](claat) directory for detailed description of the `claat` command
-line tool.
+Also, consider joining [this group](https://groups.google.com/forum/#!forum/codelab-authors),
```
Comment (nicolasgarnier) [inline README.md]: I'd go with `Also, consider joining the [codelab-authors Google Group]`...

File: README.md (hunk 1 above)
Comment (marcacohen) [reply README.md]: You mean the template doc? The formatting guides points to it but I can add a ref here too if you think that's useful.

Comment (nicolasgarnier) [reply README.md]: Yea I just saw you already pointed to it somewhere else. that's fine then :)

File: README.md (hunk 2 above)
Comment (marcacohen) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (marcacohen) [reply README.md]: Cool, can I get an approval when you get a chance?

Comment (marcacohen) [reply README.md]: Thanks for the missing protocols!

**Pull request merged**